### DISCUSSION TITLE
提升睡眠品質的方法

### DISCUSSION THREAD

**Discussion by Miinteaa** [1101-GS4719-C/Python-Community-GS4719-C#1470] [General]
期末考周來臨，為了顧好期末考試，又開始有壓縮睡眠時間、用來讀書的情況。
但睡得少也是能睡得好，以下我整理了一些對我來說有效提升睡眠品質，達到深度睡眠的方法：

### 1. 計算自己的睡眠周期
以正常情況來說，人們每晚可能有4-5次的睡眠周期循環，一次約90-100分鐘。
若以90分鐘的倍數制定睡眠時間 (EX.鬧鐘響起的時間)，起床的時候比較容易被叫醒且不容易累
（（但這個方法好像不是每個人都適用的樣子

### 2. 睡前吃B群
之前偶然看到睡前吃適量B群(B6、B1、B2、B3)可以改善失眠，所以這學期開始我都會在睡前吃兩顆B群。
個人覺得蠻有效的，且做夢的次數也有減少，可以一覺到天亮。
謹記要適量！攝取過多維生素B1、B6而會睡不著喔！

### 3. 放空、告訴大腦該睡覺了
過度繁雜的思緒會讓大腦停不下來，進而造成失眠。
這時候我會深呼吸放鬆＋想盡辦法放空大腦
大概五分鐘就會睡著了XD

大家都有哪些增進睡眠品質的方法呢? 可以分享看看！

**Comment by 1104030360** [comment]
我自己是會在睡前吃點小東西，因為血糖上升就會變得想睡覺，或是上youtube找一些放鬆的音樂聽一下（可以定時），這些都非常有效推薦給你～

**Comment by Tzuhsuannnnn** [comment]
我在高中的時候有試過用睡眠周期來設定鬧鐘，但對我來說沒什麼用...我覺得規律的睡眠時間對我來說蠻有用的，每天都差不多時間睡，差不多時間起，久而久之身體習慣之後，不用鬧鐘也可以很自然地起床。

**Comment by Aaron0810** [comment]
我個人是大概從大一到現在大三睡前都會躺在床上看個影片才睡覺的，雖然蠻多研究有說過睡前接收過多藍光會睡不著，但現在的我反而是睡前不看影片就睡不著，所以我覺得要提升睡眠品質感覺比較偏向要習慣一件睡前會做的事，向樓上有人說可以吃東西等等應該都蠻有效的。

**Comment by alstinlin** [comment]
我自己在睡前會減少滑手機的時間，因為滑手機會讓我在睡覺前一直想到剛剛看的東西，有點像沒辦法專心睡覺的意思，除此之外我習慣睡覺前洗澡，這樣會比較好入睡

**Comment by Alexgns** [comment]
我認為睡前不要使用手機效幫助睡眠，依我自己的經驗，通常一直用手機後去睡覺，常常都會在床上翻來覆去一段時間

**Comment by sumairu0727** [comment]
我通常難以睡眠都是精神太好所導致，所以我通常不會在精神好的時候爬上床睡覺，我的建議是那就讓自己做點消耗體力的事，可能騎個腳踏車洗完澡就很好睡了

**Comment by Superlan8** [comment]
我是睡眠有點過長的人，有時候一睡都睡太久，想起床又都會賴床，生理時鐘也常常因為考試而亂掉，有些人會說起床看手機會比較容易清醒，但我每次開起來又會睡著，我認為其原因有可能是我睡眠品質不好，導致睡眠時間變很長。

**Comment by pudding0803** [comment]
我是一個滿淺眠的人，所以在宿舍通常都需要耳塞才比較容易入睡

關於上述的第二點，我之前是聽說吃 B 群會讓精神變好，所以睡前如果太早吃，好像會有反效果？（精神太好睡不著）。另外，如果尿液變得很金黃，就代表那些是額外的 B 群，我吃一顆就會有了，所以我在想是不是一天吃一顆就好？

**Comment by Brownnnn78** [comment]
最近看到一個小知識是只要把專注力集中在額頭很容易就能夠睡著了，真的很迷，我也不知道為什麼真的很快就睡著。除此之外我發現吃澱粉很容易想睡，吃巧克力什麼的都沒有澱粉有用，合理解釋為什麼睡前的泡麵是必需品。最後還有適當的溫度，低溫可以幫助入睡，但這也讓我變得離開冷氣就很難以入睡。

**Comment by juliaouo** [comment]
分享個我的方法，平躺在床上，然後將眼睛閉上，先從左手開始放鬆，然後到左腳、右腳、右手，最後是頭，這樣一直循環，通常很快就睡著了，希望對失眠的人有幫助！另外睡前滑手機也會睡不太著，不過我想這是現代人的通病，可能很難改正。

**Comment by ashleyy-huang** [comment]
睡前就別使用3C產品，如電腦、手機等等的裝置，睡前可以喝一杯熱牛奶，聽聽能夠讓自己放鬆的音樂，讓自己的心寧放鬆下來，準備進入睡眠狀態，做了這些讓自己的心境的動作後，在一整晚得睡眠品質也會提升。

**Comment by JessyCheng2003** [comment]
我最近發現睡覺戴耳塞和加熱眼罩，會睡的很熟，起來後也很有精神。
